doc/proposals/openshift-4.3/operator-testing-tool.md

```diff
@@ -58,7 +58,11 @@ not in scope: Operator Developers can use the same tool to run custom, functiona
 #### Story 1 - Show pass/fail in Scorecard Output
 Today, the scorecard output shows a percentage of tests that were successful to the end user. This story is to change the scorecard output to show a pass or fail for each test that is run in the output instead of a success percentage. 
 
-The exit code of scorecard would be 0 if all tests passed. The exit code would be non-zero if tests failed. With this change scores are essentially replaced with a list of pass/fail(s).
+The exit code of scorecard would be 0 if all required tests passed. The exit code would be non-zero if any of the required tests failed. With this change scores are essentially replaced with a list of pass/fail(s).
+
+A message produced by the scorecard will show whether or not the required
+tests fully passed or not.  Tests will be labeled in such a way as to 
+specify them as required or not.
```

Comment:
This needs to be rewritten or removed in light of the new `Exit Codes` section.

Reply:
made an update to the PR to remove the exit code details from the Story and instead have them in the new Exit Code detailed design section.